solutions/telemetry/README.md

```diff
@@ -0,0 +1,10 @@
+# Telemetry App Examples
+
+This folder contains a basic temperature telemetry Kafka app, in two flavors:
+
+- `console` contains a producer/consumer Kafka app in plain Java.
+- `quarkus` contains a producer/consumer Kafka app in Quarkus.
+
+The `certs` folder is for the certificate files necessary to connect to your AMQ Streams cluster.
```

Comment:
The DynoLabs Kafka installation exports the cluster secret to a file inside the workspace. If students work always in the workspace folder, we can use relative paths to the file generated in the Kafka installation. Also that way students don't need to copy around the certificates or key/trust stores.

BTW, that opens the question of: Quarkus/Java uses key/trust stores. Should we try to create them when we install Kafka so students don't need to convert the cluster certificate?

Reply:
Not sure if that's something that they have to be aware of, at least in one of the exercises

Reply:
> BTW, that opens the question of: Quarkus/Java uses key/trust stores. Should we try to create them when we install Kafka so students don't need to convert the cluster certificate?

I think it would be nice to create it from scratch and make them use the same jks. But we should make it sth visible, or at least tell them why we should create it because this is a developer course and devs will face this a lot. In short: this is sth we should teach them/ show them.

Reply:
We can add an step after the Kafka installation (in the setup GE) to make students create the jks files.